### ISSUE TITLE
Problem with Restore pages

### ISSUE THREAD

**Issue by soyelfede** [sneakypete81/updatescanner#321]
* OS version: Mac Mojave  Version 10.14.2 
 * Firefox version: 65.0 (64-bits)
 * Update Scanner version: 4.3.1

#### Steps to reproduce the problem:
1. I updated Firefox and Update Scanner and exported the bookmarks to bookmarks.html
2. I went to Update Scanner/Menu/Restore Pages

#### What you expected to happen:
Restore pages

#### What actually happened:
![captura de pantalla 2019-01-31 a la s 11 50 29](https://user-images.githubusercontent.com/47217687/52062011-71e4af00-254e-11e9-98b8-e5182f0efbbe.png)

The window opens but it does not allow me to import the bookmarks.html file (it does not recognize it, it appears pale)
[Include a screenshot]

**Comment by AJCxZ0** [comment]
The Update Scanner bookmarks should be JSON, not HTML. Check the file and try renaming it with a `.json` extension. The default file name is `Update Scanner Backup.json`

**Comment by sneakypete81** [comment]
Note that Update Scanner 4 no longer stores its pages as bookmarks, so you should use the Backup and Restore commands provided in the Update Scanner menu:

<img width="257" alt="screenshot 2018-12-09 at 08 52 34" src="https://user-images.githubusercontent.com/38854/49695249-191d1980-fb90-11e8-8946-8675991a08d3.png">

**Comment by sneakypete81** [comment]
Thanks, that makes sense now. That upgrade flow you showed only works if you're upgrading from version 3 of the add-on (before it was updated to be a webextension in 2017). We'll be removing this text in the next release, since it's extremely unlikely that you'll want to do this now.

Once you have installed v4, you can use the backup and restore functions that I mentioned above. But you shouldn't need to unless you need to delete your profile or clear your local browsing storage.

Can you describe a bit more background about what's happened?

**Comment by soyelfede** [comment]
Hello!
If I remember correctly, the steps were:
1. I updated Firefox and then opened the US column, and the list of pages was missing.
2. I saw the following Update Scanner message:
![upgrade](https://user-images.githubusercontent.com/47217687/52147283-f9ace500-2644-11e9-9b8c-9700e4e0cb05.png)

The problem was that, instead of going to the US menu to export, I clicked on the link that says "Upgrade to Update Scanner 4" and the old page was opened, from where I followed the instructions to make the backup (in html! ).
3. Obviously, when I wanted to backup the html, and it did not work. That's how I got to this point.

I tell you that I use Firefox only to be able to use Update Scanner, since it is a key part of my work. For that reason I can change profile, delete activity, or whatever it takes ... that allows me to have US running and recover the pages I had been scanning from the old Update Scanner folder.

**Comment by sneakypete81** [comment]
Note that if you follow the instructions under the "export your bookmarks" heading in your second message above, it should produce a .json file, not an .html file. However that backup will be at least a year old, since Update Scanner can no longer access bookmarks due to changes introduced in Firefox 57.

**Comment by sneakypete81** [comment]
In any case, the main problem here is that your site list disappeared when you upgraded Firefox, for some reason.

Can you provide any more details about exactly what you did? Which version did you upgrade from and to? How did you perform the upgrade? Are you sure you didn't modify any privacy/history settings at the same time? If you go back to the previous Firefox version and upgrade again, can you reproduce the problem?

**Comment by soyelfede** [comment]
Now I see that I made an error when doing an export of the "bookmarks", when I should have made a "backup" in html.

I'm not sure what version of Firefox I had and what I updated, but I'm sure I did not modify any privacy / history settings.

Now I tried the following:
1. Remove Update Scanner
2. I installed Firefox version 64.0.2
3. I reinstalled the Update Scanner extension
4. From "show all bookmarks" I went to "Restore" bookmarks from January 3, 2019. Then I copied and a .json file was generated
5. From the US menu I went to "Restore pages" but opened a page of "Failed to restore pages".

I understand that the problem is that my list disappeared, but I do not see any way to reproduce the process.

Personally I see the option to open the bookmarks one by one and go adding them in Update Scanner, but they are more than 100 and it will take me a long time. Will there be any way to add them again, from the .html or .json that I have saved?
